CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/threadpool/ConstellationGlobalThreadPoolNGTest.java

```diff
@@ -78,7 +80,9 @@ public void testGetThreadPool() {
     @Test
     public void testGetScheduledExecutorService() {
         System.out.println("getScheduledExecutorService");
-        assertEquals(ConstellationGlobalThreadPool.getThreadPool().getScheduledExecutorService() instanceof ScheduledExecutorService, true);
+        ScheduledExecutorService s1 = ConstellationGlobalThreadPool.getThreadPool().getScheduledExecutorService();
+        ScheduledExecutorService s2 = ConstellationGlobalThreadPool.getThreadPool().getScheduledExecutorService();
+        assertEquals(s1 == s2, true);
```

Comment:
Might be worth adding a comment explaining that you're checking that a new instance isn't being created the 2nd time the function is called (based on the function implementation). Similar thing for the other tests